### ISSUE TITLE
Automatic memory profiling?

### ISSUE THREAD

**Issue by igonro** [rusmux/ipython-memory-magics#1]
Hi, I just discovered this tool and I think it's pretty good. Nice job!

I would like to ask if it would be possible to add something like [ipython-autotime](https://github.com/cpcloud/ipython-autotime).

It would be awesome to enable the extension with some magic command, and make it run on every cell.

**Comment by rusmux** [comment]
Hi, thank you for using it) 

This would be a nice feature. I think it's better to add configuration options for this. For example, to enable this mode, one would set `%config.MemoryMagics.auto = True` and `%config.MemoryMagics.auto_options = "-n -j -i 10"`. I will probably implement it within a month. 

If you have any ideas, go ahead and share it!

**Comment by igonro** [comment]
Yes! That would probably be the best option, since it would allow automatic execution but also customization. The `autotime` is that simple because there is nothing to configure.
